examples/oracle.yaml

```diff
@@ -0,0 +1,16 @@
+arch: "x86_64"
```

Comment:
No support for aarch64?

Reply:
Oddly not. Oracle publishes arm docker images, install isos, and raspberry pi image, but not arm ova/qcow vm images.